CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ and this project adheres to
 
 ### Added
 
+- Reintroduce steps (old runs) API based on GovStack spec.
```

Comment:
Could we mention that we are reintroducing them in the runs api. That way we are clear that it has no UI implications.